master/pkg/searcher/operations.go

```diff
@@ -32,12 +32,11 @@ type (
 
 // All the operation types that support serialization.
 const (
-	CreateOperation              OperationType = 0
-	TrainOperation               OperationType = 1
-	ValidateOperation            OperationType = 2
-	CloseOperation               OperationType = 4
-	ValidateAfterOperation       OperationType = 5
-	SetSearcherProgressOperation OperationType = 6
```

Comment:
this change reminds me...

Have you checked the case where a custom-search experiment was in progress before the upgrade?  The experiment can fail in an ugly way, just as long as the master doesn't panic or anything.

Reply:
good point, i added back the configs in accordance with how other searchers were removed. now, restarting a custom search experiment will just log an error.